packages/react-components/react-avatar/src/contexts/AvatarContext.ts

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+
+const avatarContext = React.createContext<AvatarContextValue | undefined>(undefined);
+
+/**
+ * Sizes for the avatar
+ */
+export type AvatarSizes = 16 | 20 | 24 | 28 | 32 | 36 | 40 | 48 | 56 | 64 | 72 | 96 | 120 | 128;
```

Comment:
Since this is now in `react-avatar`, could you leave the `AvatarSizes` type in Avatar.types.ts, and import it here instead?

Reply:
done in https://github.com/microsoft/fluentui/pull/24807/commits/d0a098deb3349820955562821260c14202d99092